gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-edit.component.html

```diff
@@ -32,9 +32,65 @@ <h3>{{ entrypointName }}</h3>
         [supportedQos]="supportedQos"
         [formControlName]="entrypoint.type + '-qos'"
       ></gio-form-qos>
+
+      <mat-card *ngIf="supportDlq" class="api-entrypoints-v4-edit__form-container__dlq">
+        <h3>Dead Letter</h3>
+        <p>
+          Define an external storage where each unsuccessfully pushed message will be stored, and configure a replay strategy to push them
+          again. You can select either an entire endpoint group or a single endpoint within a group.
+        </p>
+        <gio-form-slide-toggle class="api-entrypoints-v4-edit__form-container__dlq__toggle">
+          <div>Enable Dead Letter</div>
+          <mat-slide-toggle gioFormSlideToggle formControlName="enabledDlq" aria-label="Enable Dead Letter toggle"></mat-slide-toggle>
+        </gio-form-slide-toggle>
+
+        <mat-card *ngIf="enabledDlq" class="api-entrypoints-v4-edit__form-container__dlq">
+          <h3>Create from existing endpoint</h3>
+          <ng-container *ngIf="dlqElements.length !== 0; else noElement">
```

Comment:
Maybe better to put `dlqElements?.length > 0` because currently, if `dlqElements` is undefined there would be an error in the console and if its length is undefined (don't know if it's possible but typescript can be silly), then it would still true and try to display the table.

Reply:
done